### ISSUE TITLE
quantization EMA update clarification

### ISSUE THREAD

**Issue by wangzizhao** [schmidtdominik/LAPO#5]
Thanks for sharing this great code!
I have a question about the EMA update of the latent action codebook. In the following [line](https://github.com/schmidtdominik/LAPO/blob/main/lapo/models.py#L276-L279):
```
n = torch.sum(self.ema_count, dim=-1, keepdim=True)
self.ema_count = (
    (self.ema_count + self.epsilon) / (n + M * self.epsilon) * n
)
```
why do we normalize `ema_count` here? In the [VQVAE](https://arxiv.org/pdf/1711.00937) paper, there is no such step.

<img width="932" height="733" alt="Image" src="https://github.com/user-attachments/assets/2ea2e8b6-de22-416c-9329-aaff92bc8b8d" />

Thanks a lot!
